airflow/README.md

```diff
@@ -34,3 +34,10 @@ name_servers = tolist([
 
 Your domain should now be pointing to the airflow deployment with secure TLS
 
+# Can I use the cluster for other Opta services?
+Yup! Opta was initially designed to split between one "environment" manifest holding all the networking & cluster setup
+and many different "services" built on top of the environment. However, it is possible to have an environment and
+service in a single yaml, and to keep things simple that's what is being done here. You can still create services
+pointing to the lone yaml here as the "environment" (even though it deploys what can be considered a service as well),
+but we'd still advise you to break the current yaml to an environment and service manifest when you feel ready just
+to keep things clean.
```

Comment:
what about starting a FAQ.md at the top level to prevent duplicating these?
add the end of each example, we can drop a link to the FAQ, possibly in the reference section -> https://github.com/run-x/opta-examples/tree/b8b6e37947daecba466c37c6d7f1c6d64b983e5f/aws-lambda#references